portal/grading/views.py

```diff
@@ -17,6 +18,19 @@ class AcademyGradingView(generics.RetrieveUpdateAPIView):
     queryset = models.Grade.objects.all()
     serializer_class = serializers.GradeSerializer
 
+    def put(self, request, *args, **kwargs):
```

Comment:
`ldsagrader` is updating the grade using `PATCH`.

If you move this change to override the `update` function it should work for both `PUT` and `PATCH` as both end up calling update.
https://github.com/encode/django-rest-framework/blob/00cd4ef864a8bf6d6c90819a983017070f9f08a5/rest_framework/mixins.py#L59

Reply:
in any case can you check now @hcastilho ? I changed `update` instead and all tests pass 👌